set.mm

```diff
@@ -632273,6 +632273,103 @@ fixed reference functional determined by this vector (corresponding to
+      QUVTYQWTGUXRYQXAYQXBUOVQXQUWTUWSUXRUXQVTUVTUXCUXBEYQXDVKWFUQXPXQUVTXRUXQU
+      XPXEXPXQUVTUDZXRUXQUXPUXSDVSGZUWTYRVSGZUDXRUXQIUXPXEUXSUXTUWTUYAXPXQUXTUV
+      TDWAXFXQXPUWTUVTUXCXGXQUVTUYAXPXQUWTUWSUYAUVTUXCUXBEYQXHVKXIXCDEYRXJUOXKX
+      LXMXCDYRUTWGVLXSXTUVTUVPUXKGXNUXEVCWJUKVDUPXO $.
+  $}
+
+  ${
+    $d N x $.  $d X x y $.  $d Y x y $.
+    $( A factorial with offset is monotonely increasing (Contributed by
+       metakunt, 20-Apr-2024.) $)
+    factwoffsmonot $p |- ( ( ( X e. NN0 /\ Y e. NN0 /\ X <_ Y ) /\ N e. NN0 )
+                      -> ( ! ` ( X + N ) ) <_ ( ! ` ( Y + N ) ) ) $=
+      ( cn0 wcel cle wbr caddc co cfa cfv cc0 c1 wceq oveq2 fveq2d syl wa jca
+      cr vx vy w3a cv breq12d facwordi cc nn0cn addid1 3ad2ant1 3ad2ant2 ax-1cn
+      3brtr4d addass mp3an3 syl2an 3ad2antl1 adantr nn0addcl 3adant2 3adant1 wb
+      cmul nn0re leadd1 syl3an biimpa syl3anc 3an1rs 1re mpbid wi cn faccl nnre
+      3syl clt nngt0 0re ltle mpan mpd 1nn0 nn0ge0 nn0readdcl readdcld lemul12a
+      mpan2 1red 3expa 3adantl3 facp1 breq12 3ad2antl2 breqtrd eqbrtrrd nn0indd
+      mpbird ) BDEZCDEZBCFGZUCZBUAUDZHIZJKZCXCHIZJKZFGBLHIZJKZCLHIZJKZFGBUBUDZH
+      IZJKZCXLHIZJKZFGZBXLMHIZHIZJKZCXRHIZJKZFGBAHIZJKZCAHIZJKZFGUAUBAXCLNZXEXI
+      XGXKFYGXDXHJXCLBHOPYGXFXJJXCLCHOPUEXCXLNZXEXNXGXPFYHXDXMJXCXLBHOPYHXFXOJX
+      CXLCHOPUEXCXRNZXEXTXGYBFYIXDXSJXCXRBHOPYIXFYAJXCXRCHOPUEXCANZXEYDXGYFFYJX
+      DYCJXCABHOPYJXFYEJXCACHOPUEXBBJKZCJKZXIXKFBCUFWSWTXIYKNXAWSXHBJWSBUGEZXHB
+      NBUHZBUIQPUJWTWSXKYLNXAWTXJCJWTCUGEZXJCNCUHZCUIQPUKUMXBXLDEZRZXQRZXMMHIZJ
+      KZXTYBFYRUUAXTNZXQWSWTYQUUBXAWSYQRZYTXSJWSYMXLUGEZYTXSNZYQYNXLUHZYMUUDMUG
+      EZUUEULBXLMUNUOUPPUQURYSUUAXOMHIZJKZYBFYRUUAUUIFGZXQYRUUJXNYTVCIZXPUUHVCI
+      ZFGZYRXQYTUUHFGZRZUUMYRXQUUNWSWTYQXAXQWSWTYQUCZXARZXMDEZXODEZXMXOFGZXQUUP
+      UURXAWSYQUURWTBXLUSZUTURZUUPUUSXAWTYQUUSWSCXLUSZVAURZUUPXAUUTWSBTEWTCTEYQ
+      XLTEXAUUTVBBVDCVDXLVDBCXLVEVFVGZXMXOUFVHVIWSWTYQXAUUNUUQUUTUUNUVEUUQXMTEZ
+      XOTEZRUUTUUNVBZUUQUVFUVGUUQUURUVFUVBXMVDQUUQUUSUVGUVDXOVDQSUVFUVGMTEUVHVJ
+      XMXOMVEUOQVKVISWSWTYQUUOUUMVLZXAWSWTYQUVIUUPXNTEZLXNFGZRZXPTEZRZYTTEZLYTF
+      GZRZUUHTEZRZRUVIUUPUVNUVSUUPUVLUVMUUPUVJUVKWSYQUVJWTUUCUURXNVMEZUVJUVAXMV
+      NZXNVOVPZUTWSYQUVKWTUUCLXNVQGZUVKUUCUURUVTUWCUVAUWAXNVRVPUUCUVJUWCUVKVLZU
+      WBLTEUVJUWDVSLXNVTWAQWBUTSWTYQUVMWSWTYQRZUUSXPVMEUVMUVCXOVNXPVOVPVASUUPUV
+      QUVRUUPUVOUVPWSYQUVOWTUUCYTDEZUVOUUCUURUWFUVAUURMDEUWFWCXMMUSWHQZYTVDQUTW
+      SYQUVPWTUUCUWFUVPUWGYTWDQUTSWTYQUVRWSUWEXOMCXLWEUWEWIWFVASSXNXPYTUUHWGQWJ
+      WKWBWSWTYQXAUUJUUMVBZUUQUUAUUKNZUUIUULNZRUWHUUQUWIUWJUUQUURUWIUVBXMWLQUUQ
+      UUSUWJUVDXOWLQSUUAUUKUUIUULFWMQVIWRURYRUUIYBNZXQWTWSYQUWKXAUWEUUHYAJWTYOU
+      UDUUHYANZYQYPUUFYOUUDUUGUWLULCXLMUNUOUPPWNURWOWPWQ $.
+  $}
+
+  ${
+    $( 2x+3 is greater than x+2 for natural numbers (Contributed by metakunt,
+       20-Apr-2024.) $)
+    2xp3dxp1ge1 $p |- ( x e. NN -> 1 <_ ( ( ( 2 x. x ) + 3 ) / ( x + 2 ) ) ) $=
```

Comment:
This should also be valid for `( -u 1 (,) +oo )`, maybe the more generic version could be useful?

Reply:
Oh also, `x` does not need to be a set variable here, you could replace it with the class variable `X`